academic-services/knowledge-exploration-service/tutorial-schema-design.md

```diff
@@ -151,66 +154,65 @@ In addition to this best practice guidance, local builds have a concrete limit o
+        "DOI": "10.1145/2740908.2742839",
+        "FullTextUrl": "http://localhost/example-full-text-link-2",
+        "Title": "An Overview of Microsoft Academic Service (MAS) and Applications",
+        "AuthorAffiliations": [
+          {
+            "NormalizedAffiliationName": "microsoft",
+            "NormalizedAuthorName": "arnab sinha",
+            "AuthorName": "Arnab Sinha",
+            "Sequence": "1"
+          },
+          {
+            "NormalizedAffiliationName": "microsoft",
+            "NormalizedAuthorName": "zhihong shen",
+            "AuthorName": "Zhihong Shen",
+            "Sequence": "2"
+          },
+          ...],
+        "FieldsOfStudy": [
+          {
+            "Name": "World Wide Web",
+            "NormalizedName": "world wide web"
+          },
+          {
+            "Name": "Vertical search",
+            "NormalizedName": "vertical search"
+          },
+          ...]
+      }
+    ],
+    "timed_out": false
     }
     ```
 
-1. Run Evaluate command using the built index to verify index operations are working as expected:
+1. Verify index operations are working as expected by running the Evaluate command to retrieve specific entities:
 
     ```cmd
     kesm Evaluate --IndexFilePaths samplePrivateLibraryData.linked.kes --KesQueryExpression="Year=2020" --Attributes "Year"
     ```
 
-    The output should be
+    The command should retreieve publication entities with `Year` attibute being `2020` from the index. The output should mirror the following JSON:
```

Comment:
>The output should mirror the following JSON [](start = 104, length = 43)

I'd avoid using "mirror" and simply say "should look similar to the following JSON" instead